lib/msal-electron-proof-of-concept/samples/msal-electron-sample-app/src/Graph.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as rp from 'request-promise';
+
+export class Graph {
+    static async fetchUserData(accessToken: string): Promise<any> {
+        const graphUrl: string = `https://graph.microsoft.com/v1.0/me`;
+        const options: any = {
+            headers: {
+                Authorization: `Bearer ${accessToken}`
+            },
+            json: true
+        };
+        return rp(graphUrl, options).then((body) => {
+            return body;
+        });
```

Comment:
Can just be `return rp(graphUrl, options);`

Reply:
Done